README.md

```diff
@@ -1,4 +1,16 @@
-# test-datasets: `rnafusion`
+# ![nfcore/test-datasets](docs/images/test-datasets_logo.png)
```

Comment:
Maybe you want to add here that this is testdata for rnafusion only in case someone is mistakenly going here without knowing how we organize things :-)

Reply:
Good catch, going to change